.github/workflows/test.yml

```diff
@@ -10,4 +13,49 @@ jobs:
       - uses: actions/checkout@v4
       - uses: actions/setup-python@v5
       - run: pip install -r rockcraft_lpci_build/requirements.test.txt
-      - run: pytest rockcraft_lpci_build/tests/ -vvv -s -rP --log-cli-level=INFO
\ No newline at end of file
+      - run: pytest rockcraft_lpci_build/tests/ -vvv -s -rP --log-cli-level=INFO
+
+  chisel-wrapper:
+    name: Chisel wrapper
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Install Chisel
+        env:
+          version: v1.1.0
+        run: |
+          wget "https://github.com/canonical/chisel/releases/download/${{ env.version }}/chisel_${{ env.version }}_linux_amd64.tar.gz"
+          tar zxvf "chisel_${{ env.version }}_linux_amd64.tar.gz"
+          sudo mv chisel /usr/local/bin/
+      - name: Test chisel-wrapper
+        run: |
```

Comment:
shall we put this into a standalone test?

Here's an idea:
 - create a folder, in root, `chisel-wrapper`
 - add a symlink to the top level wrapper script (i think it would be dangerous to move it?)
 - add a tests folder, and choose whatever framework you'd like. e.g. bats+makefile?

Reply:
Yup, done. Added `tests/chisel-wrapper/runtest` file.